wremnants/datasets/datasets2016.py

```diff
@@ -25,7 +25,8 @@ def makeFilelist(paths, maxFiles=-1, format_args={}):
         filelist.extend(glob.glob(path) if path[:4] != "/eos" else buildXrdFileList(path, "eoscms.cern.ch"))
     return filelist if maxFiles < 0 else filelist[:maxFiles]
 
-def getDatasets(maxFiles=-1, filt=None, mode=None, base_path=None, nanoVersion="v9", prod_tag="TrackFitV718_NanoProdv1"):
+def getDatasets(maxFiles=-1, filt=None, mode=None, base_path=None, nanoVersion="v9", 
+        data_tag="TrackFitV722_NanoProdv1", mc_tag="TrackFitV718_NanoProdv1"):
```

Comment:
Can you revert this to TrackFitV718_NanoProdv1 for data for now? Then we can make a new PR to update it when the new samples are fully complete